### PR TITLE
LL-8154 Use BigNumber instead of BigInt

### PR DESCRIPTION
**Pull request by haammar-ledger**
https://ledgerhq.atlassian.net/browse/LL-8154

### CODE REVIEW

**Review by valpinkman — APPROVED** [review]

**Review by gre — CHANGES_REQUESTED** [review]
not passing the CI test.

**Comment by haammar-ledger** [comment]
Dead end, as `BigNumber` doesn't support `bigint` conversion.

File: packages/hw-app-btc/src/newops/psbtv2.ts [hunk 1]
```diff
@@ -559,7 +560,7 @@ function uint32LE(n: number): Buffer {
 }
 function uint64LE(n: number): Buffer {
   const b = Buffer.alloc(8);
-  b.writeBigUInt64LE(BigInt(n), 0);
+  b.writeBigUInt64LE(new BigNumber(n), 0);
```
Comment (gre) [inline packages/hw-app-btc/src/newops/psbtv2.ts:563]: is this change correct? i'm not sure writeBigUInt64LE will understand new BigNumber 🤔

Comment (gre) [reply packages/hw-app-btc/src/newops/psbtv2.ts:563]: oh sorry wrong pr

**Pull request closed without merging**